pkg/util/versions/versions_test.go

```diff
@@ -0,0 +1,36 @@
+package versions
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCalculateFCV(t *testing.T) {
```

Comment:
Can we add a test for the path that doesn't get a valid string? for example 3.0.0 -> ""

And one for an invalid semver like `3.4.5.2`?